Makefile

```diff
@@ -59,6 +59,35 @@ E2E_TEST_CONFIG ?=
 
 LDFLAGS = -ldflags "-s -w -X sigs.k8s.io/node-feature-discovery/pkg/version.version=$(VERSION) -X sigs.k8s.io/node-feature-discovery/source.pathPrefix=$(HOSTMOUNT_PREFIX)"
 
+## multi-arch build
+# required to enable buildx
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+# build with buildx
+PLATFORMS ?= linux/amd64,linux/arm64
+PROGRESS   = auto
```

Comment:
Is the `PROGRESS` variable necessary? I'd drop it. Or at least rename it to something like `BUILDX_PROGRESS`

Reply:
Yeah was thinking of this as well :)